ext/mysqli/mysqli.stub.php

```diff
@@ -0,0 +1,540 @@
+<?php
+
+class mysqli_driver
+{
+}
+
+class mysqli
+{
+    /** @return mysqli|null|false */
+    public function __construct(
+        ?string $host = null,
+        ?string $user = null,
+        ?string $password = null,
+        ?string $database = null,
+        ?int $port = null,
+        ?string $socket = null
+    );
+
+    /** @return bool|null */
+    public function autocommit(bool $mode);
+
+    /** @return bool|null */
+	public function begin_transaction(int $flags = 0, string $name = '');
+
+	/** @return bool|null */
+	public function change_user(string $user, string $password, ?string $database);
+
+	/** @return string|null|bool */
+	public function character_set_name();
+
+    /** @return bool */
+	public function close();
```

Comment:
Currently, it can return `true` or `null` (because of `MYSQLI_FETCH_RESOURCE_CONN`) so `bool|null` would be the appropriate return type. Hopefully we will be able to eliminate the `null` case from here and everywhere by throwing an exception instead.

Reply:
now that the null is gone, should this return true or bool in the signature?